linux_os/guide/services/base/service_portreserve_disabled/rule.yml

```diff
@@ -33,3 +33,10 @@ references:
 ocil: '{{{ ocil_service_disabled(service="portreserve") }}}'
 
 platform: machine
+
+template:
+    name: service_disabled
+    vars:
+        mask_service: 'true'
+        packagename@rhel6: portreserve
+        servicename: portreserve
```

Comment:
The `packagename@rhel6` key is superflous because implicit behavior will be that the package name is the same as service name.

Reply:
Looking into this.

Reply:
Fixed.